src/Calculator/AboutFlyout.xaml.cpp

```diff
@@ -64,5 +64,4 @@ void AboutFlyout::SetDefaultFocus()
 
 void CalculatorApp::AboutFlyout::UserControl_Loaded(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
```

Comment:
We should probably remove the entire event handler if we aren't doing anything with it.